relay/channel/codex/adaptor_test.go

```
@@ -54,3 +60,174 @@ func TestConvertOpenAIResponsesRequestDropsPenalties(t *testing.T) {
+			accept: r.Header.Get("Accept"),
+			path:   r.URL.Path,
+		}
+		w.Header().Set("Content-Type", "text/event-stream")
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer server.Close()
+
+	gin.SetMode(gin.TestMode)
+	w := httptest.NewRecorder()
+	c, _ := gin.CreateTestContext(w)
+	c.Request = httptest.NewRequest(http.MethodPost, "/v1/responses", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)